### PR TITLE
feat: Add Account Authentication Setting

### PR DESCRIPTION
**Pull request by Danial-EP**

### CODE REVIEW

File: src/types/mutation.graphql [hunk 1]
```diff
@@ -1,4 +1,5 @@
 type Mutation {
+  accountAuthenticationSetting:AccountAuthenticationSetting
```
Comment (dazadep) [inline src/types/mutation.graphql:2]: why is this a mutation ?

File: src/datasources/AccountsDataSource.ts [hunk 2]
```diff
@@ -23,4 +23,9 @@ export class AccountsDataSource extends RESTDataSource {
         const {data} = await this.post(`/account-members/tokens`, {data: accountTokenInput});
         return data;
     }
+
+    async accountAuthenticationSetting() {
+        const {data} = await this.get(`/settings/account-authentication`);
```
Comment (dazadep) [inline src/datasources/AccountsDataSource.ts:28]: ...since we do a get here

File: src/types/mutation.graphql (hunk 1 above)
Comment (Danial-EP) [reply src/types/mutation.graphql]: thanks, I missed it.

File: src/resolvers/mutations/accounts.ts [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
 
-const generateAccountTokens = async (root, {accountTokenInput} , {dataSources}) => {
+const generateAccountTokens = (root, {accountTokenInput} , {dataSources}) => {
     try {
         return dataSources.accountsAPI.generateAccountTokens( accountTokenInput )
     } catch (e) {
         throw new UserInputError("API returned with errors.", e)
     }
 }
+const accountAuthenticationSetting = (root, {accountTokenInput} , {dataSources}) => {
```
Comment (dazadep) [inline src/resolvers/mutations/accounts.ts:10]: this file is for mutation resolvers - maybe move this to a query resolver file.

Comment (Danial-EP) [reply src/resolvers/mutations/accounts.ts]: I missed these pieces, fix it.

**Review by dazadep — APPROVED** [review]

**Pull request merged**